data/mods/Magiclysm/enchantments/Shaman.json

```
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "mind_read",
+    "type": "SPELL",
+    "name": "Mind Reading",
+    "description": "You can predict some movements of your enemies (just slowdown in fact).  If you are seeing this spell, you have debugged it in.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT", "NO_PROJECTILE" ],
+    "effect": "attack",
+    "shape": "blast",
+    "effect_str": "mind_reading",
+    "min_damage": 0,
+    "max_damage": 0,
+    "min_aoe": 7,
+    "max_aoe": 7
```

i m very confsued pls help D:
"sense other minds" sounds like a "haha ik where u r" type thing and less a "i will read ur mind and dodge everything"
```suggestion
    "description": "You can predict some movements of your enemies, allowing you to easily dodge their attacks.  If you are seeing this spell, you have debugged it in.",
    "valid_targets": [ "self" ],
    "flags": [ "SILENT", "NO_PROJECTILE" ],
    "effect": "attack",
    "effect_str: "mind_reading",
    "shape": "blast",
    "min_damage": 0,
    "max_damage": 0,
    "min_range": "1",
    "max_range": "1",
    },
```